.github/.OwlBot.yaml

```
@@ -0,0 +1,36 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+docker:
+  image: gcr.io/repo-automation-bots/owlbot-nodejs:latest
+
+deep-remove-regex:
+  - /google/cloud
+  - /docs
+  - /scripts
+  - /tests
+
+deep-preserve-regex:
+  - /nox.py
+  - /setup.py
+  - /README.rst
+  - /docs/index.rst
+
```

```suggestion
- docs/changelog.md
- docs/README.rst

```
Added 2 more files to preserve

---

Done!